### PR TITLE
Compatibility with Alpha9

### PR DESCRIPTION
**Pull request by 95CivicSi**
This update breaks BC.  

- Updated skin methods to match the new skin api.
- ~~Hard coded standard Alex skin data.~~
- Updated handling of location to use Vector3.
- Removed obsolete variables.

### CODE REVIEW

**Comment by Falkirks** [comment]
That hardcoding is really ugly. Is there no better way to get a default skin?

**Comment by 95CivicSi** [comment]
It would be possible to move the skin data to a config file and pull from there.  It's also possible to load the skin from an existing player if they are the one who uses the command.  I'll have to check other methods.

**Comment by SOF3** [comment]
Also note a potential copyright infringement by embedding it in the plugin.

**Comment by 95CivicSi** [comment]
@SOF3 I imagine you or @dktapps would be more knowledgeable about loading a default skin with the new API.  I hadn't considered the copyright problem.  If push comes to shove, we could load a custom skin.

**Comment by dktapps** [comment]
There's no need for an original skin. The original grey Specter one works fine, it just needs to be base64 encoded when assigned to SkinData. Also, only the SkinId and SkinData fields are required.

**Comment by dktapps** [comment]
Also note that verification skip is **not** supported on the release build of ALPHA9, only latest development builds.

**Comment by 95CivicSi** [comment]
@dktapps Thanks for the replies.  I had not used Specter prior to this and did not realize that the previous skin data was just generating a gray color.  That made a huge difference in understanding how to change this.  @Falkirks Commit https://github.com/Falkirks/Specter/pull/60/commits/c96e97801bf91c42073e80278219b8b753dfc196 on my branch reverts skin data to the previous setting with one change.  SkinID is now "Specter" to help eliminate future misunderstandings.

image

File: src/specter/network/SpecterInterface.php [hunk 1]
```diff
@@ -190,11 +190,8 @@ public function decodeAdditional(){
...
+			$pk->clientData["SkinId"] = base64_encode("Specter");
```
Comment (dktapps) [inline src/specter/network/SpecterInterface.php]: that's incorrect. also, what in the world happened to the formatting?

Comment (95CivicSi) [reply src/specter/network/SpecterInterface.php]: Not sure with the formatting. Just fixed it.  What's wrong with 193?

Comment (dktapps) [reply src/specter/network/SpecterInterface.php]: it's not supposed to be base64'd. Only the skin data is.

**Pull request merged**